### PR TITLE
[5.2] [Guided Tours] Better tour user state wording and explanation

### PR DESCRIPTION
**Pull request by obuisard**
### Summary of Changes

To ease user and translators understanding of the tour user states, I have added a comment and improved the wording of the warning messages.

### Testing Instructions

None.

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [x] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [x] No documentation changes for manual.joomla.org needed

### CODE REVIEW

**Comment by Kostelano** [comment]
I have tested this item :white_check_mark: successfully on fb1dbc977907f4d0e99409beace4c9d9da98c270<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/44093">issues.joomla.org/tracker/joomla-cms/44093</a>.</sub>

File: administrator/language/en-GB/com_guidedtours.ini [hunk 1]
```diff
@@ -94,7 +94,8 @@ COM_GUIDEDTOURS_TYPE_REDIRECT_URL_DESC="Enter the relative URL of the page you w
 COM_GUIDEDTOURS_TYPE_REDIRECT_URL_LABEL="Relative URL"
 COM_GUIDEDTOURS_URL_LABEL="Relative URL"
 COM_GUIDEDTOURS_URL_DESC="Enter the relative URL of the page from where you want to Start the tour, e.g administrator/index.php?option=com_guidedtours&view=tours for the tours' list page."
-COM_GUIDEDTOURS_USERSTATE_CONNECTEDONLY="Tour User state action is only for connected users."
-COM_GUIDEDTOURS_USERSTATE_STATENOTSAVED="Tour User state not saved for user %1$s tour %2$s."
-COM_GUIDEDTOURS_USERSTATE_STATESAVED="Tour User state saved for user %1$s tour %2$s."
+; 'Tour User states' are conditions in which a guided tour user leaves a tour, either by cancelling, completing or skipping a tour.
+COM_GUIDEDTOURS_USERSTATE_CONNECTEDONLY="The recording of User states for a tour is only available to connected users."
```
Comment (Quy) [inline administrator/language/en-GB/com_guidedtours.ini]: Should `connected` be `registered` or `logged in`?

Comment (obuisard) [reply administrator/language/en-GB/com_guidedtours.ini]: Sure!

**Review by tecpromotion — APPROVED** [review]

**Comment by tecpromotion** [comment]
RTC

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 3816b9212671cb545ce69f23e66ed95e217ece92<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/44093">issues.joomla.org/tracker/joomla-cms/44093</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/44093">issues.joomla.org/tracker/joomla-cms/44093</a>.</sub>

**Comment by Hackwar** [comment]
Thanks for your contribution @obuisard!

**Pull request merged**